shared/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1162,9 +1162,10 @@ Undocumented.prototype.readSite = function( query, fn ) {
 	this.wpcom.req.get( '/read/sites/' + query.site, params, fn );
 };
 
-Undocumented.prototype.readSiteFeatured = function( siteId, fn ) {
+Undocumented.prototype.readSiteFeatured = function( siteId, query, fn ) {
+	var params = omit( query, [ 'before', 'after' ] );
```

Comment:
we now have an elegant way to do this natively since ES6

```js
const { before, after, ...params } = query;
```

but you could go another step further and merge this line into the function declaration

```js
function( siteId, { before, after, ...query }, fn ) {
	debug( … );
	this.wpcom.req.get( `/read/sites/${ siteId }/featured`, query, fn );
}
```